generators/python/sdk/versions.yml

```diff
@@ -1,4 +1,11 @@
 # For unreleased changes, use unreleased.yml
+- version: 4.3.6
+  irVersion: 53
+  changelogEntry:
+    - type: fix
+      summary: |
+        Add `test_generator_cli.py`; upgrade to use latest cli version and compatibility.
```

Comment:
This is a user facing changelog -- thoughts on just talking about how readme generation is now fixed?

Reply:
That makes sense to me. Fixing.